executor/batch_point_get.go

```diff
@@ -82,7 +82,7 @@ func (e *BatchPointGetExec) initialize(ctx context.Context) error {
 		return err
 	}
 
-	txn, err := e.ctx.Txn(true)
+	txn, err := e.ctx.Txn(false)
```

Comment:
Do we need to check all these `Txn(true)` usages, seems many

Reply:
I've checked all of them. In another way, if `Txn(true)`, it prefers to return an error when txn is invalid, but all the tests passed.